docs/asciidoc/faq.adoc

```diff
@@ -85,3 +86,24 @@ Format
 Reactor Netty adds information for the connection at the beginning of every log record (when this is possible).
 Use the id of the connection in order to extract all log records for a particular HTTP request.
 For more information see <<faq.logging-prefix>>
+
+[[faq.connection-closed]]
+== How can I debug "Connection prematurely closed BEFORE response"?
+By default, Reactor Netty clients use a connection pooling. When a connection is acquired from the connection pool, it is checked whether it is still opened.
+However, the connection can be closed at any time after the acquisition. There are many reasons that can cause connection to be closed.
+In most cases the setup might not be like: the client directly sends to the server, but there might be many other network components between them: proxies, load balancers etc.
```

Comment:
the `setup might not be like:` part of this sentence is unclear. not like what? there are two `:` in this sentence, which makes me think it was meant to be two separate sentences?

Reply:
There are two related ideas here. I would write it as two sentences:

In most cases, the client might not send directly to the server. Instead, there might be other network components (proxies, load balancers, and so on) between them.